src/components/app/MenuButtons/Publish.css

```diff
@@ -2,21 +2,44 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.menuButtonsAbortUploadButton {
-  background-color: var(--green-70);
-  color: #fff;
+.menuButtonsShareButtonButton::before {
+  background-image: url(firefox-profiler-res/img/svg/share-dark.svg);
 }
 
-.menuButtonsShareButtonOriginal {
-  background-color: var(--green-70);
-  color: white;
+@keyframes uploadingAnimation {
+  25% {
+    opacity: 0.2;
+    transform: scale(0.75);
+  }
+
+  50% {
+    opacity: initial;
+    transform: initial;
+  }
+}
+
+.menuButtonsShareButtonButton__uploading::before {
+  animation: 1s infinite uploadingAnimation;
```

Comment:
Suggestion: This animation is kind of awkward with its easing. It would be better to have a longer cubic-bezier easing curve here, rather than the hard 25% to 50% bounce.

I'm trying to think of what other animations do here, and it's usually a spinner, which is more work, but may be nicer.

Reply:
I liked the animation but I just whipped it up quickly. I don't like spinners much in general, but I'm thinking of something else I can try (combination of translateY and small scale).

Reply:
This now changed completely.